### ISSUE TITLE
Does the Stable Diffusion AI Template work with T4 and V100?

### ISSUE THREAD

**Issue by sarmientoj24** [facebookincubator/AITemplate#15]

**Comment by antinucleon** [comment]
No. We have some kernels rely on `cp.async`, which is a SM80 feature.

**Comment by kennethgoodman** [comment]
I was able to get it to run on a T4, but the results are terrible, maybe because of ^? It does run 2x as fast though

https://colab.research.google.com/gist/kennethgoodman/f42df87eda4bf427c3d3731f73e43b6f/fbaitemplate-stable-diffusion.ipynb

**Comment by antinucleon** [comment]
I just disabled T4 support: https://github.com/facebookincubator/AITemplate/pull/18
